ext/sqlite3/tests/sqlite3_41_prepared_stmt_params.phpt

```diff
@@ -0,0 +1,49 @@
+--TEST--
+SQLite3::prepare and execute with parameters as array
+--EXTENSIONS--
+sqlite3
+--FILE--
+<?php
+
+require_once(__DIR__ . '/new_db.inc');
+define('TIMENOW', time());
+
+echo "Creating Table\n";
+var_dump($db->exec('CREATE TABLE test (time INTEGER, id STRING)'));
+
+echo "INSERT into table\n";
+var_dump($db->exec("INSERT INTO test (time, id) VALUES (" . TIMENOW . ", 'a')"));
+var_dump($db->exec("INSERT INTO test (time, id) VALUES (" . TIMENOW . ", 'b')"));
+
+echo "SELECTING results\n";
+$stmt = $db->prepare("SELECT * FROM test WHERE id = ? ORDER BY id ASC");
+$foo = 'a';
+echo "BINDING Value\n";
+$results = $stmt->execute([$foo]);
```

Comment:
it would be useful to have tests for zero (when the array is empty) and multiple parameters too in order to test the `if` and the `for`